soroban-env-host/Cargo.toml

```diff
@@ -62,7 +62,7 @@ sha3 = "=0.10.8"
 # stellar-core ultimately ships. This is a temporary workaround until the next
 # protocol release when we can safely bump the minimum curve25519-dalek version
 # in stellar-core as well.
-curve25519-dalek = { version = ">=4.1.1, <=4.1.2", default-features = false, features = ["digest"]}
+curve25519-dalek = { version = ">=4.1.1, <=4.1.3", default-features = false, features = ["digest"]}
```

Comment:
Actually, can you just remove the upper boundary? We've also removed it for ed25519; I'm not sure if it provides any real value (especially now that ed25519 version is unrestricted)

Reply:
Ok, we will need to pin it to 4.1.3 after core is updated anyway. Removed the upper bound.